pkg/assets/bindata.go

```diff
@@ -98,7 +98,7 @@ func dashboardsApicastServicesJsonTpl() (*asset, error) {
 		return nil, err
 	}
 
-	info := bindataFileInfo{name: "dashboards/apicast-services.json.tpl", size: 17479, mode: os.FileMode(436), modTime: time.Unix(1613987062, 0)}
+	info := bindataFileInfo{name: "dashboards/apicast-services.json.tpl", size: 17479, mode: os.FileMode(420), modTime: time.Unix(1614032948, 0)}
```

Comment:
I guess that's not important, but it is weird to change the file mode, maybe because of using MacOs instead of Linux?

Reply:
🍎  stuff probably.